libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/MapboxSpeechPlayer.java

```diff
@@ -44,14 +43,14 @@
    * Construct an instance of {@link MapboxSpeechPlayer}
    *
    * @param context     to setup the caches
-   * @param locale      for which language
+   * @param language    for which language
    * @param accessToken a valid Mapbox access token
    */
-  MapboxSpeechPlayer(Context context, Locale locale, String accessToken) {
+  MapboxSpeechPlayer(Context context, String language, String accessToken) {
     setupCaches(context);
     instructionQueue = new ConcurrentLinkedQueue();
     voiceInstructionLoader = VoiceInstructionLoader.builder()
-      .language(locale.toString())
+      .language(language)
```

Comment:
💯 I believe this was the source of some of the `Locale` related voice bugs we were seeing